### PR TITLE
Add system APIs for build own facade

### PR DESCRIPTION
**Pull request by Thiyan**

### CODE REVIEW

File: en/docs/fhir/guides/metadata-api.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+!!! note
+        Supported FHIR version is 4.0.1.
+
+## Setup and run locally
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/metadata-api.md]: Should have ballerina tab

File: en/docs/fhir/guides/metadata-api.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+This service provides implementation of FHIR Metadata API. This implements 
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+!!! note
+        Supported FHIR version is 4.0.1.
+
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/metadata-api.md:13]: Let's add the note of implementation flavours

{!includes/bal-mi-note.md!}

File: en/docs/fhir/guides/smart-api.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+of a FHIR server.
+
+SMART defines a discovery document available at `.well-known/smart-configuration` relative to a FHIR Server Base URL, 
+allowing clients to learn the authorization endpoint URLs and features a server supports. This information helps clients to 
+direct authorization requests to the right endpoint, and helps clients construct an authorization request that the server 
+can support.
+
+!!! notes
+        Supported FHIR version is 4.0.1.
+
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/smart-api.md:16]: Do the same as in https://github.com/wso2/open-healthcare-docs/pull/33/files#r1733971870 and add the ballerina tab

**Comment by sameeragunarathne** [comment]
Sign the CLA

File: en/docs/fhir/guides/metadata-api.md (hunk 1 above)
Comment (Thiyan) [reply en/docs/fhir/guides/metadata-api.md]: Added

File: en/docs/fhir/guides/metadata-api.md (hunk 2 above)
Comment (Thiyan) [reply en/docs/fhir/guides/metadata-api.md:13]: Added

File: en/docs/fhir/guides/smart-api.md (hunk 3 above)
Comment (Thiyan) [reply en/docs/fhir/guides/smart-api.md:16]: Added

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**